hatchery/config.go

```diff
@@ -60,6 +61,16 @@ type SidecarContainer struct {
 	LifecyclePreStop []string          `json:"lifecycle-pre-stop"`
 }
 
+// LicenseInfo contains configuration for Gen3 supplied licenses.
+type LicenseInfo struct {
+	LicenseUserMapsTable string `json:"gen3-license-user-maps-dynamodb-table"`
+	LicenseType          string `json:"license-type"`
+	MaxLicenseIds        int    `json:"max-license-ids"`
+	G3autoName           string `json:"g3auto-name"`
+	G3autoKey            string `json:"g3auto-key"`
+	FilePath             string `json:"file-path"`
```

Comment:
probably need to add some validation to the values in above 3 lines as well in the codes that is using them

Reply:
A license config validation has been added. It runs through all containers at hatchery set up. The validation is also run on a container license config when the container is launched.